django-stubs/test/utils.pyi

```diff
@@ -145,6 +145,7 @@ def dependency_ordered(
 def get_unique_databases_and_mirrors(
     aliases: Optional[Set[str]] = ...,
 ) -> Tuple[Dict[_Signature, _TestDatabase], Dict[str, Any]]: ...
+def setup_databases(**kwargs: Any) -> List[Tuple[BaseDatabaseWrapper, str, bool]]: ...
```

Comment:
It does not look correct: https://github.com/django/django/blob/3.2/django/test/utils.py#L159-L160

Reply:
Updated.